main.js

```diff
@@ -1,68 +1,72 @@
 //buttons
 var rmButton = document.querySelector(".receive-message");
-
-
+var clearBtn = document.querySelector(".clear-button");
 //mantra image
-var mantraImg = document.querySelector(".mantra-bell")
-
+var mantraImg = document.querySelector(".mantra-bell");
 //text to print
 var text = document.querySelector(".text")
-
-
-
+var messageBox = document.querySelector(".message-box");
+//Loading Container
+var loadingBox = document.querySelector(".loading-box");
+//Body to change background
+var body = document.querySelector(".body-blue-to-yellow-bg");
 // objet that stores arrays
-
 var messages = {
   affirmations: affirmations,
   mantras: mantras,
 };
-
-
-
 //event listener
-rmButton.addEventListener("click", fixingStuff)
-text.addEventListener("click", function() {
-  returnPage();
-  reloadPage();
-})
+rmButton.addEventListener("click", fixingStuff);
+clearBtn.addEventListener("click", returnPage);
 
 // functions
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 };
 
 function findRandomMessage(selected) {
-  // var selected = document.querySelector('input[type="radio"]:checked')
   var randomIndex = getRandomIndex(messages[selected.value])
   text.innerHTML = messages[selected.value][randomIndex]
   revealMessage();
 };
 
-// need conditional in findRandomMessage
-//OR new function that is a conditional
-
-//guard clause:
-// 1)  what's going on
-// 2) example below:
-
 function fixingStuff() {
   var selected = document.querySelector('input[type="radio"]:checked')
   if(!selected) alert("Please Select Affirmation Or Mantra!")
   findRandomMessage(selected)
-}
-
-
+  changeBackgroundColor(selected);
+};
 
 function revealMessage() {
   mantraImg.classList.add("hidden")
-  text.classList.remove("hidden")
+  loadingBox.classList.remove("hidden")
+  messageBox.classList.add("hidden")
+  setTimeout(() => {
+    console.log('inside timeout')
+    loadingBox.classList.add("hidden")
+    messageBox.classList.remove("hidden")
+  }, 3000);
 };
 
 function returnPage() {
+  var selected = document.querySelector('input[type="radio"]:checked')
+  selected.checked = false
   mantraImg.classList.remove("hidden")
-  text.classList.add("hidden")
-}
+  messageBox.classList.add("hidden")
+};
 
-function reloadPage() {
```

Comment:
Really good! Window page loads are "expensive" operations. Removing this will speed up the site and improve the user's experience.